astropy/io/fits/hdu/table.py

```diff
@@ -27,13 +27,13 @@
 from ..fitsrec import FITS_rec, _get_recarray_field, _has_unicode_fields
 from ..header import Header, _pad_length
 from ..util import _is_int, _str_to_num
-
```

Comment:
Any reason why you removed that blank line. I think this should seperate the `fits`-imports from the general astropy imports.